JSTests/stress/temporal-plaintime.js

```diff
@@ -156,23 +156,17 @@ shouldBe(String(Temporal.PlainTime.from('2007-01-09 03:24:30[u-ca=japanese]')),
       nanosecond: 205
     });
     shouldBe(String(time), `19:39:09.068346205`);
-
-    // This is spec bug. Currently this throws an error. But possibly this should not throw an error.
-    // Tracked in https://github.com/tc39/proposal-temporal/issues/1803.
-    shouldThrow(() => {
-        Temporal.PlainTime.from({ hour: 19, minute: 39, second: 9 });
```

Comment:
Don’t we want to test that this does not throw an error, or is this case covered elsehwere?

Reply:
It's covered by the other test changes in this file, since the crux is that we shouldn't need to provide explicit zeroes.